src/index.html

```diff
@@ -11,6 +11,29 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+      data-qa="big"
+    >
+      <input
+        type="text"
+        name="search"
+        placeholder="Try &#8221Los Angeles&#8221"
+        data-qa="keypress"
+        class="searchlineBig searchline">
```

Comment:
5 point on [checklist](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md)

fix all issues from [checklist](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md) and I will check all code 👌

Reply:
fixed